### ISSUE TITLE
How to create pytorch_model.bin file?

### ISSUE THREAD

**Issue by mayanks43** [PiotrNawrot/nanoT5#30]
Hello, Thank you for creating nanoT5. I am trying to fine-tune a nanoT5 pre-trained model. After training model for 2^16 steps with default settings, the script seems to checkpoint and save a few files such as model.safetensors,  optimizer.bin, random_states_0.pkl, scheduler.bin. How do I create the pytorch_model.bin file for finetuning?

**Comment by mayanks43** [comment]
Ok, I was able to resolve this. Looks like huggingface now generates model.safetensors instead of pytorch_model.bin. To get the old behavior, I had to fix the versions of the packages accelerate and transformers to the versions given in the assets/env_dump/pip_freeze.txt file. Leaving this around for posterity.
